### PR TITLE
Another avg_weight solution

### PR DESCRIPTION
**Pull request by hutuxian**
Just for reference

### CODE REVIEW

File: paddlebox/avg_weight_better.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                                              [3],[4]], dtype='int64'), [[3, 2]], fluid.CPUPlace()),
+    'label': np.array([[1], [0]], dtype='float32')
+    }
+
+fetch_list = [avg_cost]
+# 执行训练（模拟10个minibatch），指定输入，获取输出
+for i in range(10):
+
+    # copy weight to avg_weight
+    for e in avg_list:
```
Comment (gavin1332) [inline paddlebox/avg_weight_better.py:93]: 不是每个minibatch都拷贝，是每个pass拷贝

**Review by gavin1332 — CHANGES_REQUESTED** [review]

File: paddlebox/avg_weight_better.py (hunk 1 above)
Comment (hutuxian) [reply paddlebox/avg_weight_better.py:93]: Just example，will add comment to show it more clear

Comment (hutuxian) [reply paddlebox/avg_weight_better.py:83]: update done

**Review by gavin1332 — APPROVED** [review]
LGTM

**Pull request merged**